ui/src/components/friends/friends_list/mod.rs

```diff
@@ -121,6 +115,23 @@ pub fn Friends(cx: Scope) -> Element {
                             logger::error(&format!("failed to block friend: {}", e));
                         }
                     }
+                    ChanCmd::RemoveConversation(recipient) => {
+                        let (tx, rx) = oneshot::channel::<Result<(), warp::error::Error>>();
+                        warp_cmd_tx
+                            .send(WarpCmd::RayGun(RayGunCmd::Remove2PersonConv {
```

Comment:
Could possibly be a check to determine what to send since group conversation being deleted would be broadcasted under the same event while this would only work if its a direct conversation (with only 2 recipients)

Reply:
I made a separate enumeration for removing direct conversations. i don't see a way to delete just the conversation at the moment - just a way to hide the chat. maybe this is good enough for now.